src/Tribe/Views/V2/Assets.php

```diff
@@ -650,6 +650,23 @@ public function should_enqueue_single_event_styles() {
 	 * @return boolean
 	 */
 	public function should_enqueue_single_event_block_editor_styles() {
+		/**
+		 * Checks whether the page is being viewed in Elementor preview mode.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool Should the assets be enqueued.
+		 */
+		if (
```

Comment:
This is fine - but if we're going to be doing this more we should consider creating a function for it. We do something similar in https://github.com/the-events-calendar/events-pro/blob/master/src/Tribe/Integrations/Manager.php#L125